packages/blade/src/components/Icons/LoaderIcon/LoaderIcon.native.test.tsx

```diff
@@ -0,0 +1,11 @@
+import renderWithTheme from '../../../_helpers/testing/renderWithTheme.native';
```

Comment:
Do we need LoaderIcon now since we have the SpinnerIcon? Remove it if not needed